apps/webstore/src/app/shared/components/modals/family-tree-template-modal/family-tree-template-modal.component.html

```diff
@@ -1,6 +1,12 @@
 <div data-cy="family-tree-template-modal">
     <div class="modal-header">
         <h4 class="modal-title" i18n="@@select-a-template">Select a template</h4>
+        <div data-cy="family-tree-template-close-button" class="tc-button-square" (click)="activeModal.close()">
```

Comment:
Is the family tree template modal supposed to look like this with all the extra blank space at the bottom?
<img width="1800" alt="image" src="https://user-images.githubusercontent.com/22862227/182427534-18544c3f-eb06-4e58-aca3-4ff2b7276a44.png">